### ISSUE TITLE
Fix failing accessibility test

### ISSUE THREAD

**Issue by TimOsahenru** [BlackPythonDevs/blackpythondevs.github.io#654]
Fix the failing accessibility test on this PR https://github.com/BlackPythonDevs/blackpythondevs.github.io/pull/651
